src/routes/local.ts

```diff
@@ -15,41 +16,74 @@ const PROPS_TO_SEND_IN_RESPOSNE = [
   "purpose",
 ];
 
+router.get("/two-line-elements/:id", (req, res, next) => {
+  const id = req.params.id
```

Comment:
I think that it's a bit weird you keep all these middle ware routes in the same file as the final destination. Why would one keep it as a middleware if you intend to keep them coupled.